tests/test_oml/test_models/test_models_creation.py

```diff
@@ -1,36 +1,50 @@
 from pathlib import Path
-from typing import Any, Dict
+from typing import Any, Dict, Optional
 
 import pytest
 import torch
 
 from oml.interfaces.models import IExtractor
 from oml.models.resnet import ResnetExtractor
+from oml.models.vit.clip import CLIP_MODELS, ViTCLIPExtractor
 from oml.models.vit.vit import ViTExtractor
 
 vit_args = {"normalise_features": False, "use_multi_scale": False, "strict_load": True}
-
+vit_clip_args = {"normalise_features": False}
 resnet_args = {"normalise_features": True, "gem_p": 7.0, "remove_fc": False, "strict_load": True}
 
 
 @pytest.mark.parametrize(
-    "constructor,args,default_arch", [(ViTExtractor, vit_args, "vits16"), (ResnetExtractor, resnet_args, "resnet50")]
+    "constructor,args,default_arch",
+    [
+        (ViTExtractor, vit_args, "vits16"),
+        (ViTCLIPExtractor, vit_clip_args, None),
```

Comment:
the same as the comment above: not sure about the purpose of having None as a valid architecture parameter